src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/CoreWindowExtension.cs

```diff
@@ -66,14 +66,16 @@ private void Run()
 
 			var timeval = stackalloc IntPtr[2];
 
-			foreach (var f in Directory.GetFiles("/dev/input", "event*"))
-			{
-				if (this.Log().IsEnabled(LogLevel.Debug))
+			if (Directory.Exists("/dev/input")) {
```

Comment:
```suggestion
			if (Directory.Exists("/dev/input")) 
			{
```

Reply:
@jeromelaban done, thanks for the review!